packages/harpguru-core/src/components/harp-guru-page/harp-guru-page.tsx

```diff
@@ -8,6 +8,9 @@ import type { ReactElement } from 'react'
 import { ToggleBufferFlusher } from '../toggle-buffer-flusher'
 import { NotifyOfScale } from '../notify-of-scale'
 import { NotifyOfQuizQuestion } from '../notify-of-quiz-question'
+import { NewStateLogger } from '../new-state-logger'
+import { NewHarpStrataStateCallback } from '../new-harpstrata-state-callback'
+import { NewColumnBoundsStateCallback } from '../new-columnbounds-state-callback'
```

Comment:
Should do some tests with these components, possibly add more to do that, and then remove them.

Reply:
I have seen massive improvements in the UI directly so this testing is less important. However it might be interesting to raise an investigation just to check that *all* of the things are behaving as expected.

Reply:
Raised as techdebt in https://github.com/js-jslog/harpguru/issues/120#issuecomment-938452028